feature/tunnel/otg_tests/tunnel_interface_based_ipv4_gre_encapsulation_test/README.md

```diff
@@ -0,0 +1,84 @@
+# TUN-1.3: Interface based IPv4 GRE Encapsulation
+
+## Summary
+
+Validate Interface based Ipv4 GRE Tunnel Config.
+
+## Procedure
+
+- Validate the GRE configuration
+  -  GRE Tunnel interfaces configuration options
+     -  Tunnel source
+     -  Tunnel source should be able to configure with unnumbered interface address
+     -  Tunnel Destination
+     -  Directly on Tunnel interface or Tunnel Group
+- Configure such 2000 tunnel interfaces
+- Configure static route with IPv4 NH pointing to Tunnel interfaces
+- Send 1000 IPv4 flows from tester on ATE-PORT1 connected to DUT-PORT1
+- Enable the packet capture on ATE ingress port to verify the GRE header of encapped traffic
+- Verify the tunnel interfaces counters to confirm the traffic encapsulation
+- After encapsulation, traffic should be load balanced/hash to all available L3 ECMP or LAG or combination of both features
+- Validate system for:
+  - TODO: Health-1.1
+  - TODO: No feature related error or drop counters incrementing, discussion with vendors required to highlight additional fields to monitor based on implementation and architecture
+
```

Comment:
This README is missing the required canonical OC as specified in the [README template](https://github.com/openconfig/featureprofiles/blob/main/doc/test-requirements-template.md).